From 67db93118ffa8d925ddaa355c97f476b00d16d24 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Mon, 11 Apr 2016 15:07:49 +0300
Subject: [PATCH 0162/1345] fix: mvpp2x: mv_pp2x_rx_fifo_init compilation
 warning fix

commit  0318fc993a9f74cb7d72d1ea77ca204a6a1748d5 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- Remove unused mv_pp2x_rx_fifo_init function

Change-Id: I9076ba7f459ecd79324e533e3e79dc811a1599ee
Reviewed-on: http://vgitil04.il.marvell.com:8080/28925
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c |   17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
index 6093176..099c289 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
@@ -4037,23 +4037,6 @@ static void mv_pp2x_conf_mbus_windows(const struct mbus_dram_target_info *dram,
 	mv_pp2x_write(hw, MVPP2_BASE_ADDR_ENABLE, win_enable);
 }
 
-/* Initialize Rx FIFO's */
-static void mv_pp2x_rx_fifo_init(struct mv_pp2x_hw *hw)
-{
-	int port;
-
-	for (port = 0; port < MVPP2_MAX_PORTS; port++) {
-		mv_pp2x_write(hw, MVPP2_RX_DATA_FIFO_SIZE_REG(port),
-			    MVPP2_RX_FIFO_PORT_DATA_SIZE);
-		mv_pp2x_write(hw, MVPP2_RX_ATTR_FIFO_SIZE_REG(port),
-			    MVPP2_RX_FIFO_PORT_ATTR_SIZE);
-	}
-
-	mv_pp2x_write(hw, MVPP2_RX_MIN_PKT_SIZE_REG,
-		    MVPP2_RX_FIFO_PORT_MIN_PKT);
-	mv_pp2x_write(hw, MVPP2_RX_FIFO_INIT_REG, 0x1);
-}
-
 /* Initialize network controller common part HW */
 static int mv_pp2x_init(struct platform_device *pdev, struct mv_pp2x *priv)
 {
-- 
1.7.9.5

